libnavui-maps/build.gradle

```diff
@@ -38,7 +38,7 @@ android {
 }
 
 dependencies {
-    api project(":libnavui-base")
+    implementation project(":libnavui-shield")
```

Comment:
This could potentially be considered a breaking change if someone was referencing one of the classes from base via `maps` module. Minor, but doesn't hurt to keep it in.

Reply:
This is the dependency block in `libnavui-shield`
```
dependencies {
    api project(":libnavui-base")
    implementation project(":libnavui-resources")
```
I think just changing `implementation project(":libnavui-shield")` to `api project(":libnavui-shield")` should be good enough